Doc/library/pickle.rst

```diff
@@ -366,6 +366,19 @@ The :mod:`pickle` module exports two classes, :class:`Pickler` and
 
       Use :func:`pickletools.optimize` if you need more compact pickles.
 
+   .. method:: reducer_override(self, obj)
+
+      Special reducer that can be defined in ``Pickler`` subclasses. This
+      method has priority over any reducer in the ``dispatch_table``.  It
+      should conform to the same interface as a :meth:`__reduce__` method, and
+      can optionally return ``NotImplemented`` to fallback on
+      ``dispatch_table``-registered reducers to pickle ``obj``.
+
+      For a detailed example on how to use ``reducer_override``, see:
+      `Subclassing the Pickler class`_
```

Comment:
You should be able to use:

..., see: :ref:`reducer_override`.

If I am not mistaken.